sdk/tests/tutorials/properties/test_derived_properties.py

```diff
@@ -0,0 +1,132 @@
+import unittest
+
+import json
+
+import lusid
+
+import lusid.models as models
+from utilities import InstrumentLoader
+from utilities import TestDataUtilities
+
+class DerivedPropertyTests(unittest.TestCase):
+
+
+    @classmethod
+    def setUpClass(cls):
+        # create a configured API client
+        api_client = TestDataUtilities.api_client()
+        cls.scopes_api = lusid.ScopesApi(api_client)
+        cls.property_definitions_api = lusid.PropertyDefinitionsApi(api_client)
+        cls.instruments_api = lusid.InstrumentsApi(api_client)
+        # load instruments from InstrumentLoader
+        instrument_loader = InstrumentLoader(cls.instruments_api)
+        cls.instrument_ids = instrument_loader.load_instruments()
+
+    def create_ratings_property(self, figi, fitch_value=None, moodys_value=None):
+
+        ratings = ["Fitch", "Moodys"]
+
+        for rating in ratings:
+            property_definition = models.CreatePropertyDefinitionRequest(
+                domain="Instrument",
+                scope="Test-Demo",
+                code="{}Rating".format(rating),
+                display_name="{}Rating".format(rating),
+                data_type_id=lusid.ResourceId(
+                    scope="system", code="number"))
+
+            try:
+                # create property definition
+                self.property_definitions_api.create_property_definition(
+                    create_property_definition_request=property_definition)
+            except lusid.ApiException as e:
+                if json.loads(e.body)["name"] == "PropertyAlreadyExists":
+                    pass
+
+        properties = {"Instrument/Test-Demo/FitchRating": fitch_value, "Instrument/Test-Demo/MoodysRating": moodys_value}
+
+        # upsert property definition
+        for key in properties:
+            if properties[key] is not None:
+                property_request = [
+                    models.UpsertInstrumentPropertyRequest(
+                        identifier_type="Figi",
+                        identifier=figi,
+                        properties=[
+                            models.ModelProperty(key=key,
+                                                 value=models.PropertyValue(
+                                                     metric_value=models.MetricValue(value=properties[key]))
+                                                 )]
+                    )]
+
+                response = self.instruments_api.upsert_instruments_properties(
+                    upsert_instrument_property_request=property_request)
+
+    def get_instruments_with_deriv_prop(self, Figi):
```

Comment:
Maybe change the name to:
`get_instruments_with_derived_prop`
We might want to be a bit more verbose as 'deriv' can sometimes be shorthand for 'derivatives'